downloads/swan-lake-release-notes/swan-lake-2201.8.3/RELEASE_NOTE.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: ballerina-left-nav-release-notes
+title: Swan Lake Update 8 (2201.8.3) 
+permalink: /downloads/swan-lake-release-notes/2201.8.3/
+active: 2201.8.3
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201.8.3
+    - /downloads/swan-lake-release-notes/2201.8.3/
+    - /downloads/swan-lake-release-notes/2201.8.3-swan-lake/
+    - /downloads/swan-lake-release-notes/2201.8.3-swan-lake
+---
+
+## Overview of Ballerina Swan Lake Update 8 (2201.8.3)
+
+<em>Swan Lake Update 8 (2201.8.3) is the third patch release of Ballerina 2201.8.0 (Swan Lake Update 8) and it includes a new set of bug fixes to the runtime and developer tooling.</em>
+
+## Update Ballerina
+
+Run the command below to update your current Ballerina installation directly to 2201.8.3 by using the [Ballerina Update Tool](/learn/update-tool/) as follows.
+
+```
+$ bal dist pull 2201.8.3
+```
+
+## Install Ballerina
+
+If you have not installed Ballerina, download the [installers](/downloads/#swanlake) to install.
+
+## Runtime updates
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.8.3 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.8.3+label%3AType%2FBug+is%3Aclosed+label%3ATeam%2FjBallerina).
+
+## Developer tools updates
+
+### Bug fixes
+To view bug fixes, see the GitHub milestone for Swan Lake Update 8 (2201.8.3) of the repositories below.
+
+- [Dev Tools](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.8.3+label%3ATeam%2FDevTools+label%3AType%2FBug+is%3Aclosed)
```

Comment:
```suggestion
- [Dev Tools](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.8.3+label%3ATeam%2FDevTools+label%3AType%2FBug+is%3Aclosed)
```
This is also empty.

Reply:
I think there is no difference with suggested changes

Reply:
The link is empty.

Reply:
It is because the issues are not closed since master PRs are not merged

Reply:
Is it affect to the release note?

Reply:
We can't have empty links on the release note. The release note should be released together with the release, which means, after completing the fixes and closing the issues.

Reply:
We have to create a separate issues for 8.x then
I will work on that